pkg/test/integration/server.go

```diff
@@ -0,0 +1,202 @@
+/*
+Copyright 2019 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"time"
+
+	"github.com/pkg/errors"
+	apiextensionsv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	"k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+
+	// Allow auth to cloud providers
```

Comment:
I opened the https://godoc.org/k8s.io/client-go/plugin/pkg/client/auth trying to find more context on what it means to allow auth to cloud providers by importing a package, but could not find any. Perhaps it's worth adding a little more explanation here?

Reply:
It just allows you to authenticate to managed k8s offerings on a variety of providers. The GCP plugin has a pretty good doc [here](https://github.com/kubernetes/client-go/blob/8d0e6f1b7b780ef8c8f81800d504509cd1f68e9f/plugin/pkg/client/auth/gcp/gcp.go#L58).

Reply:
That's good context, thanks! Would you mind capturing it in the comment in case any future readers are also unclear?

Reply:
I'd still prefer a clearer explanation here, but will merge to unblock this work.

Reply:
Oops yes meant to add a more descriptive comment, will follow up!